types/geojson/index.d.ts

```diff
@@ -76,15 +76,16 @@ export type GeoJSON = Geometry | Feature | FeatureCollection;
  * Geometry object.
  * https://tools.ietf.org/html/rfc7946#section-3
  */
-export type Geometry = Point | MultiPoint | LineString | MultiLineString | Polygon | MultiPolygon | GeometryCollection;
+export type Geometry = BasicGeometry | GeometryCollection;
+export type BasicGeometry = Point | MultiPoint | LineString | MultiLineString | Polygon | MultiPolygon;
```

Comment:
use the name `BasicGeometry` to avoid a breaking change.

otherwise, we could have:
```ts
export type Geometry = Point | MultiPoint | LineString | MultiLineString | Polygon | MultiPolygon;
export type GeometryObject = Geometry | GeometryCollection;
```